noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/common.nr

```diff
@@ -287,16 +288,15 @@ pub fn update_end_values(
         }
     }
     public_inputs.end.new_l2_to_l1_msgs.extend_from_bounded_vec(new_l2_to_l1_msgs_to_insert);
-
+    
     // logs hashes
     // See the following thread if not clear:
     // https://discourse.aztec.network/t/proposal-forcing-the-sequencer-to-actually-submit-data-to-l1/426
-    let previous_encrypted_logs_hash = public_inputs.end.encrypted_logs_hash;
-    let current_encrypted_logs_hash = private_call_public_inputs.encrypted_logs_hash;
-    public_inputs.end.encrypted_logs_hash = compute_logs_hash(previous_encrypted_logs_hash,current_encrypted_logs_hash);
-    let previous_unencrypted_logs_hash = public_inputs.end.unencrypted_logs_hash;
-    let current_unencrypted_logs_hash = private_call_public_inputs.unencrypted_logs_hash;
-    public_inputs.end.unencrypted_logs_hash = compute_logs_hash(previous_unencrypted_logs_hash,current_unencrypted_logs_hash);
+
+    // TODO(Miranda): Once we use actual logs, silo them here
```

Comment:
I think it makes sense to first separate note preimage logs from other logs, then work on siloing them here, since they may be different structs?

Reply:
Ok great, I will remove this comment! Is this the same for `public_tail`? In the public kernels messages etc are currently siloed in their `propogate` call (in setup/app_logic)

Reply:
Yes everything in private and public!